synapse/lib/certdir.py

```diff
@@ -193,288 +229,356 @@ def __init__(self, path=None):
         for p in path:
             self.addCertPath(p)
 
-    def addCertPath(self, *path):
+    def addCertPath(self, *path: str):
 
         fullpath = s_common.genpath(*path)
         self.pathrefs[fullpath] += 1
 
         if self.pathrefs[fullpath] == 1:
             self.certdirs.append(fullpath)
 
-    def delCertPath(self, *path):
+    def delCertPath(self, *path: str):
         fullpath = s_common.genpath(*path)
         self.pathrefs[fullpath] -= 1
         if self.pathrefs[fullpath] <= 0:
             self.certdirs.remove(fullpath)
             self.pathrefs.pop(fullpath, None)
 
-    def genCaCert(self, name, signas=None, outp=None, save=True):
+    def genCaCert(self, name: str,
+                  signas: StrOrNone = None,
+                  outp: OutPutOrNone =None,
```

Comment:
Should these be `= None` or `=None`? Seems to be a mix of spacing in use

Reply:
`= None` is preferred.